src/coreclr/nativeaot/Runtime/GCMemoryHelpers.cpp

```diff
@@ -21,91 +21,17 @@
 //
 // USAGE:  The caller is responsible for hoisting any null reference exceptions to a place where the hardware exception
 //         can be properly translated to a managed exception.
-COOP_PINVOKE_CDECL_HELPER(void *, RhpInitMultibyte, (void * mem, int c, size_t size))
+COOP_PINVOKE_CDECL_HELPER(void *, RhpInitMultibyte, (void * mem, size_t size))
```

Comment:
Change the comment and/or the name to indicate that the buffer can contain GC references?

Reply:
Right. I am also tempted to move this to a managed helper as well, but I do not see a lot of reasons.
This is only used when unboxing a nullable null.

Reply:
I'll move this to managed when/if we have a caller that could realistically need to zero something larger than `BulkMoveWithWriteBarrierChunk`.